modules/accredited_representative_portal/lib/tasks/seed/staging_seed/methods.rb

```diff
@@ -0,0 +1,198 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  module StagingSeeds
+    RequestOptions = Struct.new(
+      :org, :rep, :claimant, :resolution_cycle,
+      :resolved_time, :unresolved_time, :totals,
+      keyword_init: true
+    )
+
+    module RequestMethods
+      def create_claimants(count = 10)
+        # just grab users to be claimants
+        UserAccount.limit(count).to_a
```

Comment:
Thinking of potential issues with this. Only issue vector I can think of would be whether we can successfully establish POA for these people in upstream services.

Potentially we can cross that bridge when we get there.